### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Properties_Blob Container Properties tab_Screen Reader: Voice over is announcing as Clickable text for the elements present inside the "Propertied Pane".

### ISSUE THREAD

**Issue by keerthiduvvuri** [microsoft/AzureStorageExplorer#7607]
### Preflight Checklist

- [X] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [X] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [X] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.32.1(96)

### Regression From

NO

### Architecture

x64

### Storage Explorer Build Number

20231114.10

### Platform

macOS

### OS Version

MAC Version 13.6.1

### GitHub Tags:
#A11yMAS;#A11yTCS;#MAC;#DesktopApp;#A11ySev2;#BM_AzureToolsStorageExplorer_MAC_Dec2023;#Azure Tools Storage Explorer;#FTP;#Voiceover;#MAS1.3.1;#Info and Relationship;#AILimited;#Element:Textbox;

### Environment Details:
Application: Microsoft Azure Storage Explore
version: 1.32.1 (96)
Screen Reader: Voice Over

### Steps to Reproduce

1. Launch Storage Explorer
2. Tab Navigate to "Open Connect dialog" pane and Sign into Azure.
3. Tab Navigate to "Manage Accounts" pane and select the subscriptions you will use. Once selected, click on "Apply" button.
4. Select "Blob Containers" of the selected subscription.
5. Tab Navigate to "Properties" Tab present on left bottom of the screen and hit enter.
6. Turn on Voice over and Observe that Voice over is announcing as Clickable text for the elements present inside the "Propertied Pane".

### Actual Experience

Voice over is announcing as Clickable text for the elements present inside the "Propertied Pane".

### Expected Experience

Voice over should not be announced as Clickable text for the elements present inside the "Propertied Pane", if it is Clickable then there should be shortcut key announced to the users.

### User Impact:
Screen reader users will face difficulty if Voice over is announcing as Clickable text for the elements present inside the "Propertied Pane".

### Attachments:
![A11y_Azure Tools Storage Explorer_Properties_Blob Container Properties tab_Screen Reader](https://github.com/microsoft/AzureStorageExplorer/assets/98728920/1f9cfa4c-71d4-4d96-bd07-54eee3cccbeb)

**Comment by MRayermannMSFT** [comment]
@keerthiduvvuri this issue has been raised in the past and decided as a non-issue. Please see here: https://github.com/microsoft/AzureStorageExplorer/issues/5767

**Comment by keerthiduvvuri** [comment]
As per discussion in the Office hour call Closing this bug as this is expected behavior of Voice over.
